AEPCore/Sources/core/MobileCore+Configuration.swift

```diff
@@ -95,4 +95,15 @@ public extension MobileCore {
 
         MobileCore.dispatch(event: event)
     }
+
+    /// Clears all identifiers from the SDK and generates a new Experience Cloud ID (ECID).
```

Comment:
How much detail should we add here?  Should we state specifically that it clears the Edge Identities and generates a new Edge Identity ECID?
"Clears all identities from Edge Identity and Edge extensions and generates a new Experience Cloud ID (ECID) in the Edge Identity extension." ?

Reply:
I'm a bit on the fence... I don't know if we should mention edge directly? If it is so niche to only Edge use cases then maybe this API belongs in Edge...